lib/atoms/h1/h1.component.js

```diff
@@ -1,15 +1,11 @@
-angular
-    .module('lnPatterns')
-    .component('lnAH1', lnAH1);
-
 var lnAH1 = {
-    restrict: 'E',
-    replace: true,
     templateUrl: 'lnPatterns/atoms/h1/template.html',
     bindings: {
-        title: '<',
-        class: '<'
+        title: '@',
```

Comment:
Why use @ instead of < ? I saw @chrisweight was using < . I'm happy to use whichever works best, so long as we're consistent.